From 91ab89ee7f30960af9ae38cf2fd7adf248ece9e1 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Fri, 2 Nov 2012 20:45:58 +0100
Subject: [PATCH] Don't depend on not existing upstart script for fontconfig

The fontconfig_cache upstart script does not exist in openwebos so we should really not
rely on it to be sure the browserserver is started on boot.

Open-webOS-DCO-1.0-Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 browserserver |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/browserserver b/browserserver
index 012922a..c9c04f8 100644
--- a/browserserver
+++ b/browserserver
@@ -1,7 +1,7 @@
 # BrowserServer
 # Copy me into /etc/event.d
 
-start on stopped fontconfig_cache
+start on stopped finish
 
 # Stop when the Software Update tool is about to install an update. The tool
 # uses upstart to restart jobs when the installation is complete.
-- 
1.7.10.4

